Makefile

```diff
@@ -1,6 +1,8 @@
 DIST := dist
 IMPORT := code.gitea.io/gitea
 
+export PATH := $(HOME)/go/bin:node_modules/.bin:$(PATH)
```

Comment:
I would at least do`$GOPATH/bin`. Assuming it's `$HOME/go/bin` is bound to fail, especially on servers.

Reply:
Actually `$(go env GOPATH)/bin` might be even better.